src/libs/styled/useColorScheme.tsx

```diff
@@ -1,38 +1,55 @@
-import { ColorSchemeName, Appearance } from 'react-native'
+import { useColorScheme as useSystemColorScheme } from 'react-native'
 
+import { useFeatureFlag } from 'libs/firebase/firestore/featureFlags/useFeatureFlag'
+import { RemoteStoreFeatureFlags } from 'libs/firebase/firestore/types'
 import { createStore } from 'libs/store/createStore'
 
-export type ColorScheme = 'dark' | 'light'
+export enum ColorSchemeEnum {
```

Comment:
https://github.com/pass-culture/pass-culture-app-native/blob/b2bf57dd45588692fd5c9d75af438237f5a912b6/doc/development/naming.md#key-points

```suggestion
export enum ColorScheme {
```